core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -157,9 +164,10 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
     */
   @Test
   def testProduceConsume {
+    AclCommand.main(topicBrokerReadAclArgs)
     AclCommand.main(produceAclArgs)
     AclCommand.main(consumeAclArgs)
-    TestUtils.waitAndVerifyAcls(TopicReadAcl ++ TopicWriteAcl ++ TopicDescribeAcl, servers.head.apis.authorizer.get, topicResource)
+    TestUtils.waitAndVerifyAcls(TopicReadAcl ++ TopicWriteAcl ++ TopicDescribeAcl ++ TopicBrokerReadAcl, servers.head.apis.authorizer.get, topicResource)
     TestUtils.waitAndVerifyAcls(GroupReadAcl, servers.head.apis.authorizer.get, groupResource)
```

Comment:
Shouldn't we be checking that all the servers have the ACL, not just the first one? That applies to all the lines above.